src/Core/hco_calc_mod.F90

```diff
@@ -904,7 +906,11 @@ SUBROUTINE Get_Current_Emissions( HcoState, BaseDct,   nI,   nJ,           &
           RETURN
        ENDIF
     ELSE
+#ifdef MODEL_GEOS
+       LevDct1_Unit = HCO_EMISL_LEV
```

Comment:
Could you give background on why this is needed for GEOS but not GC-Classic/GCHP.

Reply:
The first bracket is simply to make HEMCO work in an ESMF environment even if the PBLHEIGHT field is not defined. There are HEMCO applications in GEOS that don't require PBLHEIGHT to be defined.

The second update is to make sure that variable LevDct1_Unit is always defined. This should also go into the standard code, I think

Reply:
Okay, sounds good.